framework/src/service/ServiceListeners.cpp

```diff
@@ -446,6 +446,15 @@ void ServiceListeners::GetMatchingServiceListeners(const ServiceEvent& evt,
       const LDAPExpr& ldapExpr = sse.GetLDAPExpr();
       if (ldapExpr.IsNull() || ldapExpr.Evaluate(props, false)) {
         set.insert(sse);
+      } else {
+        if (evt.GetType() == ServiceEvent::SERVICE_REGISTERED) {
+          const std::string msg =
+            sse.GetBundleContext().GetBundle().GetSymbolicName() +
+            " reference LDAP filter doesn't match " +
+            ref.GetBundle().GetSymbolicName() + " component's properties.\n";
+          coreCtx->logger->Log(
+            cppmicroservices::logservice::SeverityLevel::LOG_DEBUG, msg);
```

Comment:
I would like to see the debug message be more descriptive about the reason why someone is seeing the debug message. From a users standpoint, I don't understand what this debug message is telling me.

Reply:
added verbose details